tests/plugins/hold_htlcs.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+"""Plugin that holds on to HTLCs indefinitely.
```

Comment:
Nitpick: it looks to me the plugin holds on to the HTLC for only 10 seconds, not indefinitely.